### ISSUE TITLE
Move to Ascii Doc documentation colocated to each source code repo

### ISSUE THREAD

**Issue by salaboy** [Activiti/Activiti#2159]
Gitbook is not working as we wanted to initially. The documentation gets out dated really fast and the organisation of the docs is not easy. The sync mechanism for publishing fails and we want to make sure that we add our docs as part of our release process. 

Using Ascii Doc and maven plugins to generate and aggregate the documentation seems to be the popular option for large open source projects where having multi repositories and aggregation is required: 

Ie Spring Cloud: https://github.com/spring-cloud/spring-cloud-kubernetes/tree/master/docs

**Comment by mteodori** [comment]
@rallegre @erdemedeiros wdyt? I guess we experienced sync issues a few times

**Comment by erdemedeiros** [comment]
@mteodori, yes, we have some sync issues between git repo and gitbook. We should probably have a look at the different possible solutions and see which one is more adapted to the project.
